docs/Configuration.md

```diff
@@ -21,7 +21,7 @@ You can add these configurations into spark-defaults.conf to enable or disable t
 | spark.plugins                                                | To load Gluten's components by Spark's plug-in loader                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | org.apache.gluten.GlutenPlugin                       |
 | spark.shuffle.manager                                        | To turn on Gluten Columnar Shuffle Plugin                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | org.apache.spark.shuffle.sort.ColumnarShuffleManager |
-| spark.gluten.enabled                                         | Enable Gluten, default is true. Just an experimental property. Recommend to enable/disable Gluten through the setting for `spark.plugins`.                                                                                                                                                                                                                                                                                                                                                                                                | true                                                 |
+| spark.gluten.enabled                                         | Enable Gluten at runtime, default is true. It fallbacks to vanilla Spark for all query plans. Recommend to enable/disable Gluten through the setting for `spark.plugins`.                                                                                                                                                                                                                                                                                                                                                                                                | true                                                 |
```

Comment:
To be clearer "It fallbacks to vanilla Spark for all query plans if set to false."